scripts/update-models.ts

```
@@ -0,0 +1,103 @@
+  const res = await axios.get('https://api.anthropic.com/v1/models', {
+    headers: {
+      'x-api-key': apiKey,
+      'anthropic-version': '2023-06-01',
+    },
+  });
+  const ids: string[] = (res.data.data || []).map((m: any) => m.id || m.name);
+  writeModelFile('src/claude/ModelClaude.ts', 'ModelClaude', ids);
+}
+
+async function updateDeepSeek(apiKey: string) {
+  const client = new OpenAI({ apiKey, baseURL: 'https://api.deepseek.com', dangerouslyAllowBrowser: true });
```

ESLint

[error] Replace `·apiKey,·baseURL:·'https://api.deepseek.com',·dangerouslyAllowBrowser:·true` with `⏎····apiKey,⏎····baseURL:·'https://api.deepseek.com',⏎····dangerouslyAllowBrowser:·true,⏎·`

(prettier/prettier)